### ISSUE TITLE
Line breaks are not working as expected

### ISSUE THREAD

**Issue by kunal-chowdhury** [OpenLiveWriter/OpenLiveWriter#636]
Earlier, when we added line breaks in the 'Edit' tab, it was adding `<p>&#160;</p>` in the HTML code. But now, it's adding blank space. The major problem happens when you design your post, navigate to HTML source and return back to the editor, it looses all line breaks. Irritating!!

**Comment by lakadpilipinas** [comment]
having the same issue.

**Comment by jritmeijer-muhimbi** [comment]
Yep, same here, very annoying. In the most recent release  (0.6.2.0) switching between Edit and Source almost always introduces a lot of formatting changes. I have to restyle my document a good 4 or 5 times before being able to publish it.

I have tried soft breaks (SHFT-ENTER) followed by spaces, hard breaks followed by spaces, no matter what it switches back.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
